### PR TITLE
Fix some django deprecation warnings

### PR DESCRIPTION
**Pull request by diox**
Needed for #5763, does not fix all warnings yet though, that depends on #1573 being fixed first.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,8 @@
 [tool:pytest]
 addopts = -vs --reuse-db --showlocals --tb=short
+# filterwarnings =
+#    error
+#    ignore::DeprecationWarning:csp
```
Comment (EnTeQuAk) [inline setup.cfg]: Can this be removed?

File: src/olympia/addons/models.py [hunk 2]
```diff
@@ -374,6 +374,14 @@ class Addon(OnChangeMixin, ModelBase):
...
     class Meta:
         db_table = 'addons'
+        index_together = [
+            ['weekly_downloads', 'type'],
+            ['created', 'type'],
+            ['bayesian_rating', 'type'],
+            ['last_updated', 'type'],
+            ['average_daily_users', 'type'],
+            ['type', 'status', 'disabled_by_user'],
+        ]
```
Comment (EnTeQuAk) [inline src/olympia/addons/models.py]: ❤️

File: src/olympia/addons/models.py [hunk 3]
```diff
@@ -374,6 +374,14 @@ class Addon(OnChangeMixin, ModelBase):
 
     class Meta:
         db_table = 'addons'
+        index_together = [
+            ['weekly_downloads', 'type'],
+            ['created', 'type'],
+            ['bayesian_rating', 'type'],
+            ['last_updated', 'type'],
+            ['average_daily_users', 'type'],
```
Comment (EnTeQuAk) [inline src/olympia/addons/models.py]: this can easily be tuples though, actually should be I think.

**Comment by EnTeQuAk** [comment]
django-jinja has landed, lgtm!

File: src/olympia/addons/models.py (hunk 3 above)
Comment (diox) [inline src/olympia/addons/models.py]: I thought so as well, but django docs example is a list: https://docs.djangoproject.com/en/1.11/ref/models/options/#index-together

**Pull request merged**